apis/apiextensions/v1alpha1/zz_generated.environment_config_types.go

```diff
@@ -24,7 +24,9 @@ import (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:storageversion
 // +genclient
+// +kubebuilder:storageversion
```

Comment:
can you help me understand why there are two `+kubebuilder:storageversion` directives, and if their ordering/placement is meaningful? 😇

Reply:
Uhm, no, I think I modified the script and forgot to drop the old one

Reply:
ah okay 😅 - so one of these will be removed if you just run `make generate` again? 🤔

Reply:
Fixed, unfortunately there is a bit of guessing involved into appending the storage version marker in the right place. I missed a $ and the selector was matching both `genclient` and `genclient:nonNamespaced`. We should keep an eye on it when reviewing future PRs.